packages/circuits/tests/multiplier.test.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * @jest-environment node
+ */
+/* eslint-disable jest/no-expect-resolves */
+/* eslint-disable @typescript-eslint/camelcase */
+/* eslint-disable jest/no-hooks */
+
+import { genSNARK, SNARKResult } from '~utils/snark'
+import {
+  checkPhase1Setup,
+  compileCircuit,
+  getArtifactPaths,
+  phase2Setup,
+  prepareArtifactsDirectory,
+} from './helper'
+
+const fileName = 'multiplier.test.circom'
+const artifacts = getArtifactPaths(fileName)
+const { wasm, finalZkey, vk } = artifacts
+
+describe('multiplier.test.circom', () => {
```

Comment:
@weijiekoh Using shelljs, I could construct an efficient dev envrionment! Thank you so much!
@kobigurk Finally setup the efficient testing env here. Please refer this file to test some circuit vulnerabilites.